src/journal/journald-server.c

```diff
@@ -827,8 +827,8 @@ static bool shall_try_append_again(JournalFile *f, int r) {
                 log_ratelimit_warning(JOURNALD_LOG_RATELIMIT, "%s: Journal file has been deleted, rotating.", f->path);
                 return true;
 
-        case -ETXTBSY:         /* Journal file is from the future */
-                log_ratelimit_warning(JOURNALD_LOG_RATELIMIT, "%s: Journal file is from the future, rotating.", f->path);
+        case -ETXTBSY:         /* Time jumped backwards relative to last journal entry */
+                log_ratelimit_warning(JOURNALD_LOG_RATELIMIT, "%s: Time jumped backwards relative to last journal entry, rotating.", f->path);
```

Comment:
i'd really use a different error code, and leave ETXTBSY thing for the "future file"